server/routerlicious/packages/lambdas/package.json

```diff
@@ -54,6 +54,7 @@
     "@fluidframework/server-services-client": "^0.1029.0",
     "@fluidframework/server-services-core": "^0.1029.0",
     "@fluidframework/server-services-telemetry": "^0.1029.0",
+    "@fluidframework/server-services-utils": "^0.1029.0",
```

Comment:
Discussed offline, but tracking it here: we can't add services-utils to lambdas because lambdas are isomorphic and services-utils is not